features/de.cau.cs.kieler.klighd.view.feature/feature.xml

```diff
@@ -222,6 +222,7 @@ litigation.
 
    <requires>
       <import feature="de.cau.cs.kieler.klighd.feature" version="1.0.0.v20200217" match="compatible"/>
+      <import feature="de.cau.cs.kieler.klighd.feature" version="1.0.0.v20200217" match="compatible"/>
```

Comment:
Why did you duplicate this line?
Also (unrelated to this PR in perticular), why does this requirement have a version constraint anyway? There should be none as it is part of the same project (and the other features do not have any version constraint on the klighd.feature either)